src/test/resources/features/15-attachment/attachments.feature

```diff
@@ -0,0 +1,19 @@
+ Feature: Attachments Management
+
+   Background:
+     Given a user logs in, searches a patient and visits Attachment page 
+     And a user clicks on Attachment link
+     Then the system loads Attachment page
+     And a user attaches a file
```

Comment:
must every scenario require attaching a file?

Reply:
After taking again a use case, have realised its specific on one scenario. Have refactored the code base in the PR